asctb-api/src/routes/ontology-lookup.ts

```diff
@@ -17,18 +22,19 @@ export function setupOntologyLookupRoutes(app: Express): void {
     switch (ontologyCode) {
     case OntologyCode.HGNC: {
       const response = await axios.get(buildHGNCApiUrl(termId), {
-        headers: { 'Content-Type': 'application/json' }
-      }
-      );
+        headers: { 'Content-Type': 'application/json' },
+      });
       if (response.status === 200 && response.data) {
         const firstResult = response.data.response.docs[0];
-
         res.send({
+          extraLinks: {
+            'Uniprot Link': buildUniprotLink(firstResult.uniprot_ids[0]),
+            'Entrez Link': buildEntrezLink(firstResult.entrez_id),
+          },
           label: firstResult.symbol,
           link: buildHGNCLink(firstResult.hgnc_id),
-          description: firstResult.name ? firstResult.name : ''
+          description: firstResult.name ? firstResult.name : '',
```

Comment:
no trailing commas

Reply:
trailing commas were added by prettier extension in VS code. I will remove it .